File: Houseclub/src/main/java/me/grishka/houseclub/fragments/BaseToolbarFragment.java
package me.grishka.houseclub.fragments;

import android.content.res.Configuration;
import android.os.Bundle;
import android.view.View;

import me.grishka.appkit.fragments.ToolbarFragment;

public abstract class BaseToolbarFragment extends ToolbarFragment {

    @Override
    public boolean wantsLightStatusBar() {
        return true;
    }

    @Override
    public boolean wantsLightNavigationBar() {
        return true;
    }

    @Override
    public void onViewCreated(View view, Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        getToolbar().setElevation(0);
    }

    @Override
    public void onConfigurationChanged(Configuration newConfig) {
        super.onConfigurationChanged(newConfig);
        getToolbar().setElevation(0);
    }
}

File: Houseclub/src/main/java/me/grishka/houseclub/fragments/BaseToolbarFragment.kt
package me.grishka.houseclub.fragments

import android.content.res.Configuration
import android.os.Bundle
import android.view.View
import me.grishka.appkit.fragments.ToolbarFragment

abstract class BaseToolbarFragment : ToolbarFragment() {
    override fun wantsLightStatusBar(): Boolean {
        return true
    }

    override fun wantsLightNavigationBar(): Boolean {
        return true
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        toolbar.elevation = 0f
    }

    override fun onConfigurationChanged(newConfig: Configuration) {
        super.onConfigurationChanged(newConfig)
        toolbar.elevation = 0f
    }
}